platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/consensus/ConsensusTests.java

```diff
@@ -50,9 +79,12 @@ public static void initConfig() {
     void orderInvarianceTests(final ConsensusTestParams params) {
         ConsensusTestRunner.create()
                 .setTest(ConsensusTestDefinitions::orderInvarianceTests)
-                .setParams(params)
+                .setParams(modifyParams(params))
                 .setIterations(NUM_ITER)
                 .run();
+        assertMarkerFile(ConsensusImpl.NO_SUPER_MAJORITY_MARKER_FILE, false);
```

Comment:
These 3 lines are repeated verbatim in each test, consider making a private utility method to assert that these marker files are all absent. Or add an `afterEach` method

Reply:
added afterEach method

Reply:
Although I am wondering now if it indicates while method the after each is called after.